fallout/jobs/job_types/legion/explorer.dm

```diff
@@ -0,0 +1,26 @@
+/datum/job/f13/legion/explorer
+	title = "Explorer"
```

Comment:
"Legion" pre-fix at the beginning would be nice, Explorer is a bit of a vague title that I can see some people getting confused about.